src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const { slug = '' } = useParams();
+
+  const loadPeople = async () => {
+    try {
+      const peopleFromServer = await getPeople();
+
+      setPeople(peopleFromServer);
+    } catch {
+      setIsError(true);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    loadPeople();
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+          {isLoading
+            ? <Loader />
+            : (
+              <>
+                {isError && (
+                  <p data-cy="peopleLoadingError" className="has-text-danger">
+                    Something went wrong
+                  </p>
+                )}
+
+                {people.length === 0 && (
+                  <p data-cy="noPeopleMessage">
+                    There are no people on the server
+                  </p>
+                )}
+
+                <PeopleTable people={people} id={slug} />
```

Comment:
Slug - is not id

Reply:
And why do you need to pass it in props? just use `useParams` inside `PeopleTable`